pontoon/base/templates/django/account/base.html

```diff
@@ -0,0 +1 @@
+../base.html
```

Comment:
Why do we need this file and `pontoon/base/templates/django/socialaccount/base.html`?